pkg/service/manifest/storage/storage.go

```diff
@@ -14,11 +14,25 @@ type StoredManifest struct {
 	Issuer   string                      `json:"issuer"`
 }
 
+type StoredApplication struct {
+	ID          string                         `json:"id"`
+	Application manifest.CredentialApplication `json:"application"`
+	ManifestID  string                         `json:"manifest_id"`
```

Comment:
let's keep camel case for json https://github.com/TBD54566975/ssi-service/blob/main/pkg/service/did/storage/storage.go#L15

`manifestId`